### ISSUE TITLE
Добавьте функцию убрать просмотренные посты с ленты

### ISSUE THREAD

**Issue by icynstant** [vtosters/lite#1548]
Если я посмотрел пост, так пусть при обновлении страницы или при повторном заходе в приложение он не будет появляться.

В оф. вк такое есть для "Сначала интересных", у вас же оно отсутствует, для "Сначала новых" и раздела "Интересное" тоже.

**Comment by Wperdex** [comment]
@icynstant 
![IMG_4263](https://github.com/vtosters/lite/assets/102802937/7a44ad8f-0027-4609-bb95-3d8d553914be)

**Comment by icynstant** [comment]
> @icynstant 
> ![IMG_4263](https://github.com/vtosters/lite/assets/102802937/7a44ad8f-0027-4609-bb95-3d8d553914be)
> 

И? Во первых, это не решает проблему того, что ваши интересные новости работают не так как в оф. вк, неправильно. Во вторых, у меня и так стоят сначала новые. Пришлось поставить, потому что надоело смотреть на старые посты, раньше же всегда пользовался интересными новостями. В третьих, как это вообще помогает моему запросу функции? Все и так знают что можно выбирать либо сначала новые, либо сначала интересные, я же прошу чтобы был пункт в настройках с не появлением просмотренных постов. И для сначала новых, и для сначала интересных, и для вкладки интересное, в идеале так вообще выборочно.

**Comment by Wperdex** [comment]
@icynstant ааа, так ты хочешь интересную ленту из оф клиента… Тогда пользуйся оф клиентом, кек.
У версии ВК, на которой основан Лайт устаревшие алгоритмы новостей, поэтому интересная лента работает кое-как. За их замену на актуальные в ближайшее (и не только) время никто не возьмётся.

![IMG_8002](https://github.com/vtosters/lite/assets/102802937/bd256490-6549-4fd8-a3a8-c15d0106ebe1)

**Comment by icynstant** [comment]
> @icynstant ааа, так ты хочешь **интересную ленту** из оф клиента… Тогда пользуйся оф клиентом, кек.
> У версии ВК, на которой основан Лайт (5.56.1) — устаревшие алгоритмы ленты, поэтому **интересная** работает кое-как и обновляется по другому. За замену алгоритмов на актуальные в ближайшее _(и не только)_ время никто не возьмётся.
> 
> _**Апрельский скрин**_
> ![IMG_8002](https://github.com/vtosters/lite/assets/102802937/b05e41f7-f39d-44e5-9bce-ff9596e86da3)
> 

Поэтому я и прошу чтобы была функция что я описал, вы вообще читали о какой функции я просил? Я не говорил про алгоритмы, я говорил про функцию, которую как мне кажется добавить намного легче чем лазить в алгоритмах, просто чтобы то, что я уже посмотрел больше не показывалось

**Comment by Wperdex** [comment]
@icynstant да, я читал и да, то, о чём Вы просите — такое же лазание в алгоритмах. Это настройка ленты на удаление определённых постов из рекомендаций. Повторюсь, лазить в ленте никто не будет. Как максимум — из-за лени, нехватки времени и более важных фич на повестке, а как минимум — из-за страха сломать всё.

**Comment by icynstant** [comment]
> @icynstant да, я читал и да, то, о чём Вы просите — такое же лазание в алгоритмах. Это настройка ленты на удаление определённых постов из рекомендаций. Повторюсь, лазить в ленте никто не будет. Как максимум — из-за лени, нехватки времени и более важных фич на повестке, а как минимум — из-за страха сломать всё.

А, понял, раз это одно и то же тогда извиняюсь, спасибо за инфу
